api/command_help_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package api
+
+import (
+	"github.com/mattermost/platform/model"
+	"github.com/mattermost/platform/utils"
+	"testing"
+)
+
+func TestHelpCommand(t *testing.T) {
+	th := Setup().InitBasic()
+	Client := th.BasicClient
+	channel := th.BasicChannel
+
+	*utils.Cfg.SupportSettings.HelpLink = ""
```

Comment:
Can you add a `defer` statement above this to return the setting to its previous value after the test? `utils.Cfg` persists between tests and its written back to config.json, so we want to keep it the same between runs. The look something like this (ripped from `api/emoji_test.go` if you want to see more examples):
```go
	EnableCustomEmoji := *utils.Cfg.ServiceSettings.EnableCustomEmoji
	defer func() {
		*utils.Cfg.ServiceSettings.EnableCustomEmoji = EnableCustomEmoji
	}()
	*utils.Cfg.ServiceSettings.EnableCustomEmoji = true
```